core/bones/credential.py

```diff
@@ -33,7 +35,9 @@ def unserialize(self, valuesCache, name):
         """
         return {}
 
-    def singleValueFromClient(self, value, skel, name, origData):
+    def singleValueFromClient(self, value: Any, skel: 'SkeletonInstance',
+                              bone_name: str, client_data: dict
+                              ) -> tuple[Any, list[ReadFromClientError] | None]:
         err = self.isInvalid(value)
         if not err:
```

Comment:
```suggestion
        if not (err := self.isInvalid(value)):
```
Is this better so?  @sveneberth